charts/hawkbit-update-server/values.yaml

```diff
@@ -104,8 +116,19 @@ config:
       datasource:
         username: hawkbit
         password: hawkbit
-  application:
-    spring:
-      security:
-        user:
-          name: admin
+
+
+# dependency charts config
+mysql:
+  mysqlUser: hawkbit
```

Comment:
instead of the fields `useMysql` I think would be more consistent with a `mysql.enabled` field.
What do you think?

Reply:
We could do that but would be a breaking change.

Reply:
done